certbot/account.py

```diff
@@ -117,6 +117,18 @@ def load(self, account_id):
         except KeyError:
             raise errors.AccountNotFound(account_id)
 
+class RegistrationResourceWithNewAuthzrURI(messages.RegistrationResource):
+    """A backwards-compatible RegistrationResource with a new-authz URI.
+
+       Hack: Certbot versions pre-0.11.1 expect to load
+       new_authzr_uri as part of the account. Because people
+       sometimes switch between old and new versions, we will
+       continue to write out this field for some time so older
+       clients don't crash in that scenario. Because this is a
+       temporary hack, just hardcode the known new-authz URI for
+       Let's Encrypt.
+    """
+    new_authzr_uri = jose.Field('new_authzr_uri')
```

Comment:
Will old clients ever need to actually use the saved value, or does it just crash when it tries to load it but it actually uses a different method? Because if the former, this is temporary in the sense of "until we get updated versions into all distros we support", which might happen some time after a potential change to the url.